samples/tab-device-permissions/nodejs/src/components/Tab.tsx

```diff
@@ -59,10 +62,24 @@ const Tab = () => {
         </>
       }
       {isWeb &&
+      <>
         <Segment
-          /* Component to Get/Show geo-Location */
+          /* Component to capture image in browser */
+          content={<CaptureImageDesktop />}
+        />
+        <Segment
+          /* Component to capture audio in browser */
+          content={<CaptureAudioDesktop />}
+        />
+        <Segment
+          /* Component to capture video in browser */
+          content={<CaptureVideoDesktop />}
+        />
+        <Segment
+          /* Component to Get/Show geo-Location in browser */
           content={<GetLocationDesktop />}
```

Comment:
Screenshot change in ReadMe.md doesn't contain the 

![image](https://user-images.githubusercontent.com/31851992/152580549-71b06aad-66d1-4d73-82f1-5ecb7c95347f.png)

Description still talks about how this is handled in browser.
<img width="606" alt="image" src="https://user-images.githubusercontent.com/31851992/152580620-7d886200-e7c5-4415-afba-c7ccf4c6702c.png">

Could you please capture the screenshot for browser?